packages/devtools_app/lib/src/utils.dart

```diff
@@ -201,7 +201,7 @@ typedef VoidAsyncFunction = Future<void> Function();
 
 /// A typedef to represent a function taking a single argument and with no
 /// return value.
-typedef VoidFunctionWithArg = void Function(dynamic arg);
+typedef VoidFunctionWithArg<T> = void Function(T arg);
```

Comment:
use  
`void Function(ArgType arg)` 
instead of this typedef
it is no longer and it is clearer.
Maybe mark this typedef as deprecated.

Reply:
done.